Apps/Admin/Server/Program.cs

```diff
@@ -104,6 +104,7 @@ public static async Task Main(string[] args)
 
         private static void AddModules(IServiceCollection services, IConfiguration configuration, ILogger logger, IWebHostEnvironment environment)
         {
+            ExceptionHandling.ConfigureProblemDetails(services, environment);
```

Comment:
This is a result of a merge conflict and shouldn't be included.

Reply:
Yes that shouldn't be there.  It was moved.

Reply:
Removed.